1-js/11-async/07-microtask-queue/article.md

```diff
@@ -72,7 +72,7 @@ promise.catch(err => alert('caught'));
 window.addEventListener('unhandledrejection', event => alert(event.reason));
 ```
 
-...But if we forget to add `.catch`, then, after the microtask queue is empty, the engine triggers the event:
+But if we forget to add `.catch`, then after the microtask queue is empty, the engine triggers the event:
```

Comment:
I like it better with the comma. Or, take out both of the last two commas: "`catch`, then after the microtask queue is empty the engine triggers the event:". But prefer it w/ all the commas as was.